### PR TITLE
[GH-1669] Replace power up object

### LINKED ISSUE #1669: [FEATURE] Replace power up object
## 🚀 Feature
Replace the current power up model (ref) with the new asset: https://drive.google.com/file/d/1aT5gRo0oKwZqp4yMlfUoUmxfzWsel4lu/view?usp=drive_link.

Note: the animations should remain functional regardless of the object being used.

Ref:
image

### PR DESCRIPTION
**Pull request by alfopisano**
Closes #1669

## Motivation
We need to replace the power up object with its new version 

## Summary of changes
- Replaced the old object with the new asset. 
[add new power up](https://github.com/lambdaclass/curse_of_mirra/pull/1672/commits/9851f56619ae891092f1f02a3ca2a574ab56983a)
- Deleted the now unused power up. 
[delete unused powerups and material](https://github.com/lambdaclass/curse_of_mirra/pull/1672/commits/f15d6b66a91d43d9ff991026a4824df808870efa)

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [x] I have tested the changes in another devices.
  - [x] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

File: client/Assets/Prefabs/Projectiles/HackSlingshot.prefab
Comment (BertovDev) [inline client/Assets/Prefabs/Projectiles/HackSlingshot.prefab:1]: Checkout this.

Comment (BertovDev) [inline client/Assets/Prefabs/Projectiles/HackSlingshot.prefab:1]: Checkout this.

File: client/Assets/Resources/Fonts/SourceSans3-Bold SDF.asset
Comment (BertovDev) [inline client/Assets/Resources/Fonts/SourceSans3-Bold SDF.asset:1]: And also this.

**Review by BertovDev — CHANGES_REQUESTED** [review]
Everything seem to be working well and just delete the unwanted files. 

One more thing, did you follow any reference for the scale of the prefab? if not, I will decrease it a little bit like 0.6 in each axis.

btw I like the explanation for each commit.

File: client/Assets/Prefabs/Projectiles/HackSlingshot.prefab
Comment (alfopisano) [reply client/Assets/Prefabs/Projectiles/HackSlingshot.prefab:1]: Oh sorry!! Nice catch 🙌

File: client/Assets/Resources/Fonts/SourceSans3-Bold SDF.asset
Comment (alfopisano) [reply client/Assets/Resources/Fonts/SourceSans3-Bold SDF.asset:1]: Same as above.

**Comment by alfopisano** [comment]
> Everything seem to be working well and just delete the unwanted files.
> 
> One more thing, did you follow any reference for the scale of the prefab? if not, I will decrease it a little bit like 0.6 in each axis.
> 
> btw I like the explanation for each commit.

I did not follow any reference for scaling the prefab but I agree with you, it looks a little too big in comparison with the character's size.

**Comment by alfopisano** [comment]
I tried scaling the power up on 0.6 and it looks much better now!! 

And btw sorry for all the commits, I got really messed up trying to restore the fonts and slingshot files 😅

**Comment by BertovDev** [comment]
> I tried scaling the power up on 0.6 and it looks much better now!!
> 
> And btw sorry for all the commits, I got really messed up trying to restore the fonts and slingshot files 😅

No worriess, we should merge https://github.com/lambdaclass/curse_of_mirra/pull/1667 asap

**Review by BertovDev — APPROVED** [review]
LGTM

Tested on Android S23

**Comment by AngieDutra** [comment]
I think the aura of the orbe is too big. I think the overall look would benefit from making the amokey effect of the orbe smaller

**Review by BertovDev — APPROVED** [review]
Well done, looking good to me.

Tested on Xiomi redmi note 9S and Samsung s23

**Review by AngieDutra — APPROVED** [review]
Nice! Tested in Android build

**Pull request merged**